cms/models.py

```diff
@@ -51,3 +52,23 @@ class Meta:
 
     def __str__(self):
         return f'{self.post}-{self.caption}'
+
+
+class MenuItem(models.Model):
```

Comment:
Len mi tak napadlo, ze nechceme mat moznost aj na externe menu itemy? Mozno je to blbost ale mi tak napadlo, ze co ak niekedy budeme mat nejaku docasnu stranku niekde inde ale stale na nu budeme chciet mat odkaz v menu? Mozno nieco ako odkaz na naboj, it akademiu, nejake divne prihlasovanie niekde pre dalsi grant co niekedy budeme mat...

Mozno je to odveci a nebude to nikdy treba :-D

Reply:
Veď to to podpdporuje. Ten url field by mal zvládnuť aj cudziu url. Či nie?

Reply:
Aha, vlastne ano :-D